esmvaltool/recipes/recipe_ocean_scalar_fields.yml

```diff
@@ -47,7 +48,299 @@ diagnostics:
     variables:
       mfo: # sea_water_transport_across_line
         mip: Omon
+        preprocessor: prep_timeseries_drake
+        additional_datasets:
+          - {dataset: ACCESS1-0,        project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
```

Comment:
You might want to consider adding the dataset keys that are the same for all datasets (it looks like that is everything but the dataset name?) as keys under the variable, that would really increase readability.